airflow/www/api/experimental/endpoints.py

```diff
@@ -110,3 +112,24 @@ def task_info(dag_id, task_id):
     task = dag.get_task(task_id)
     fields = {k: str(v) for k, v in vars(task).items() if not k.startswith('_')}
     return jsonify(fields)
+
+
+@api_experimental.route('/latest_runs', methods=['GET'])
+@requires_authentication
+def latest_dag_runs():
+    """Returns the latest running DagRun for each DAG formatted for the UI. """
+    from airflow.models import DagRun
+    session = settings.Session()
```

Comment:
Ah sorry. In this case however the DB access should be move to api/common/experimental. See the other examples. Or just use the provided session and don't open a session at all here.